services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -55,6 +58,24 @@ class WriteMetadataActor(override val batchSize: Int,
     dbAction.map(_ => allPutEvents.size)
   }
 
+  def sanitizeInputs(
+    metadataWriteActions: NonEmptyVector[MetadataWriteAction]
+  ): NonEmptyVector[MetadataWriteAction] =
+    metadataWriteActions.map { metadataWriteAction =>
+      val metadataEvents =
+        metadataWriteAction.events.map { event =>
+          if (event.value.isDefined && metadataKeysToClean.contains(event.key.key)) {
+            event.copy(value = Option(MetadataValue(StringUtil.cleanUtf8mb4(event.value.get.value))))
```

Comment:
Looks like we still have one unprotected `get` in here - can remove by using a `match` statement rather than `if`. 
```
match (event.value) { 
    case Some(eventVal) if metadataKeysToClean.contains(event.key.key) => event.copy(... use eventVal here)
    case _ => event
}
```